index.html

```diff
@@ -45,6 +45,106 @@ <h1>
+              </div>
+              <div class="see-project"><a href="#">See project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="tech">
+                <span>Ruby on rails</span>
+                <span>css</span>
+                <span>html</span>
+                <span>Javascript</span>
+              </div>
+              <div class="see-project"><a href="#">See project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="tech">
+                <span>Ruby on rails</span>
+                <span>css</span>
+                <span>html</span>
+                <span>Javascript</span>
+              </div>
+              <div class="see-project"><a href="#">See project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="tech">
+                <span>Ruby on rails</span>
+                <span>css</span>
+                <span>html</span>
+                <span>Javascript</span>
+              </div>
+              <div class="see-project"><a href="#">See project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="tech">
+                <span>Ruby on rails</span>
+                <span>css</span>
+                <span>html</span>
+                <span>Javascript</span>
+              </div>
+              <div class="see-project"><a href="#">See Project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="tech">
+                <span>Ruby on rails</span>
+                <span>css</span>
+                <span>html</span>
+                <span>Javascript</span>
+              </div>
+              <div class="see-project"><a href="#">See Project</a></div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="about">
+        <!-- <img src="images/illustration-about-me-mobile.svg" alt="" /> -->
+        <div class="about-content">
+          <h1>About me</h1>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <div class="resume-btn">
```

Comment:
- [ ] The button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design). Kindly add this.
![gedgsdg](https://user-images.githubusercontent.com/26767383/153494750-a3530af4-b792-4008-bbb0-f4c75d7efbaa.PNG)